backend/package.json

```diff
@@ -8,7 +8,8 @@
     "langchain": "^0.0.113",
     "openai": "^3.3.0",
     "pdf-parse": "^1.1.1",
-    "react": "^18.2.0"
+    "react": "^18.2.0",
+    "sinon": "^15.2.0"
```

Comment:
I've not heard of this, where is it used?

Reply:
Oh that was a mocking library i tried but didn't use - removing now!